resources/net.unvanquished.Unvanquished.desktop

```diff
@@ -5,8 +5,9 @@ Comment=FPS/RTS Game - Aliens vs. Humans
 Icon=unvanquished
 Terminal=false
 Type=Application
-Exec="%1/updater"
+Exec="%1/updater" %u
```

Comment:
Do you think we should do

updater <updater args> -- <unvanquished args>

This way, we can always disambiguate updater args from Unvanquished args?

Reply:
This is a very good idea.

Reply:
Done.